src/cli.rb

```diff
@@ -150,6 +150,19 @@ def run
         c.action Commands::Status
       end
 
+      command :ipmi do |c|
+        c.syntax = 'metal ipmi NODE_IDENTIFIER [options]'
+        c.summary = 'Perform ipmi commands on single or multiple machines'
+        c.description = "***VOLATILE***\n\n" \
+                    'Perform ipmi commands on single or multiple machines'
+        #c.example 'description', 'command example'
+        c.option '-g',
+          'Specifies that NODE_IDENTIFIER is the group. MUST be before NODE_IDENTIFIER'
+        c.option '-k COMMAND',
```

Comment:
Since we always need to specify a command for `metal ipmi`, it seems a nicer syntax for the command would be `metal ipmi NODE_IDENTIFIER IPMI_COMMAND [options]` - how complicated a change do you think this would be? If this is simple to do now without re-writing the whole `ipmi command` in Ruby then we could do this now, otherwise can just add a card for it.